frontend/src/components/App/App.tsx

```diff
@@ -66,6 +67,14 @@ export const App: React.FC = () => {
       </Page>
     )
 
+  if (!initialized)
+    return (
+      <Page>
+        <Header />
+        <LoadingMessage message="Starting up..." />
+      </Page>
+    )
```

Comment:
This was needed because deep links were showing a missing device screen before it could connect while the device data was loading.